tests/test_buffering.py

```diff
@@ -252,36 +252,6 @@ def test_two_heterogeneous_buffers():
     assert np.all(v.data == v1.data)
 
 
-def test_over_injection():
```

Comment:
Why removed?

Reply:
a complete mess to keep supporting, for zero usefulness in practice. Since, in practical scenarios, we have a neat separation between the modulo-buffered and the saved TimeFunctions, it's only the latters that require buffering, which means buffering doesn't need to go through non-affine loops such as those stemming from interpolation